sbnanaobj/StandardRecord/SRFlashMatch.cxx

```diff
@@ -23,9 +30,15 @@ namespace caf
   void SRFlashMatch::setDefault()
   {
     present        = false;
-    score          = -5.0;
     time           = -5.0;
-    pe             = -5.0;
+    charge_q       = -5.0;
+    light_pe       = -5.0;
+    score          = -5.0;
+    scr_y          = -5.0;
+    scr_z          = -5.0;
+    scr_rr         = -5.0;
+    scr_ratio      = -5.0;
+    // what about chargeXYZ and lightXYZ?
```

Comment:
Good question. I don't think leaving them un-defaulted is a good idea, but also (-5, -5, -5) doesn't sound right.